Polish

Schwartz10 opened the pull request:
Add some tests

Also removed the larger sync status warning in favor of something less intrusive and obstructive

Schwartz10 committed:
Polish

Add some tests

navFooh committed:
bump base-css

navFooh committed:
Remove empty column in proposal table

navFooh committed:
Update Titles.tsx

navFooh committed:
Use PageTitle sideContent

navFooh committed:
Use PageTitle sideContent

navFooh committed:
Use PageTitle sideContent

navFooh committed:
Update detail.tsx

navFooh committed:
Linter fix

navFooh committed:
Only render "Action required" column when any proposal action is required

navFooh committed:
update snapshots

navFooh committed:
update actor overview page title

navFooh commented:
Hey, I saw that it's a recurring pattern that we add content next to the page title so I updated to `PageTitle` component to accomodate this. It also makes sure that the margins of the `h2` element are applied to the wrapper so that collapsing margins still work properly with content below.

I also noticed that the column count wasn't consistent throughout the proposal table due to conditionally adding an "Action required" cell. I made some adjustments so that the whole column is omitted when none of the proposals have an "Action required".

Also added a missing `hr` below the page title in the message detail page.

navFooh committed:
bump base-css

navFooh committed:
Allow to pass button text to search bar

navFooh committed:
Add SearchAddress component

navFooh committed:
Update SearchAddress placeholder

navFooh committed:
linter fix

Schwartz10 commented on src/components/HistoryTables/MsigProposals/ProposalRow.tsx:
```diff
@@ -67,14 +73,16 @@ type ProposalHistoryRowProps = {
   inspectingAddress?: string
-  actionRequired?: boolean
+  proposalActionRequired?: boolean
+  anyProposalActionRequired?: boolean
```
What is the difference between these two?

Schwartz10 commented on src/components/HistoryTables/MsigProposals/ProposalRowColumnTitles.tsx:
```diff
@@ -1,14 +1,24 @@
+      <th>Proposer</th>
+      <th>Value</th>
+      <th># Approvals</th>
+      {anyProposalActionRequired && <th />}
```
When would this be false?

Schwartz10 commented on src/components/HistoryTables/MsigProposals/index.tsx:
```diff
@@ -7,19 +8,43 @@ import {
+    [data?.msigPending]
+  )
+
+  const anyProposalActionRequired = useMemo<boolean>(
```
Still slightly confused at this

Schwartz10 reviewed (commented):
This mostly looks good, just not sure about the change in props for Proposal table

navFooh committed:
export SearchAddress

navFooh committed:
approve / cancel update

Remove anyProposalActionRequired

Update ProposalRow.tsx

consistent naming of approve

Update index.tsx

Update index.stories.js

Update Proposal.stories.js

Rename address to msigAddress

Do not pass approvalsUntilExecution to approve / cancel

linter fix

Fix prop type
